.github/workflows/main.yml

```diff
@@ -6,14 +6,17 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: Check files using the black formatter
-        uses: rickstaa/action-black@v1
-        id: action_black
-        with:
-          black_args: ". --check"
       - name: Install dependencies for demo Python project
+        if: always()
         run: |
           python -m pip install --upgrade pip
           pip install autoflake pytest
+      - uses: psf/black@stable
+        with:
+          version: "23.12.1"
+          options: ". --check"
       - name: autoflake
-        run: autoflake -r --in-place --remove-all-unused-imports   --ignore-init-module-imports --remove-unused-variables src
+        if: always()
+        run: |
+          autoflake -r  --remove-all-unused-imports   --ignore-init-module-imports --remove-unused-variables src
+          autoflake -rc  --remove-all-unused-imports   --ignore-init-module-imports --remove-unused-variables src
```

Comment:
what is the reason to make two calls with the only difference being the `c` flag?

Reply:
The call with -c just checks and does not print anything and the call without -c just prints and returns zero. So in order to have a failure and output I use both. But maybe there is an action available that does both in a better way. Also I would like to have a annotation in changed files like the typo check does, which doesn't work either.